members/WIL000326.yaml

```diff
@@ -35,7 +35,7 @@ contact_form:
           required: true
     - click_on:
         - value: Submit
-          selector: div#contour_form_Contact input[name='submitbtn']
+          selector: div.contourNavigation input[name='submitbtn']
```

Comment:
is there an element with an ID that you can use instead of the `.contourNavigation` class? I could potentially see a div with that class being used in multiple places on a form, which could cause this to break in the future.